packages/components/Organisms/Accordion/Accordion.vue

```diff
@@ -196,7 +204,7 @@ export default {
 
     &__button {
       @include rpl_typography_ruleset($rpl-accordion-title-ruleset);
-      color: $rpl-accordion-button-text-color;
+      @include rpl_text_color($rpl-accordion-button-text-color);
```

Comment:
Hi @alan-cole should we always use `rpl_text_color ` instead of `color` in Ripple?

Reply:
Basically, yes. Any text that is part of the printout should use rpl_text_color, as it includes additional @media print color definition.